// basscss-radium/index.js
var exports = module.exports =
{
	body: {
		margin: 0,
		fontFamily: '\'Helvetica Neue\', Helvetica, sans-serif',
		lineHeight: 1.5,
		fontSize: '100%',
		color: '#111',
		backgroundColor: '#fff'
	},
	img: {
		maxWidth: '100%'
	},
	svg: {
		maxHeight: '100%'
	},
	input: {
		fontFamily: 'inherit',
		fontSize: '1rem',
		boxSizing: 'border-box',
		marginTop: 0,
		marginBottom: 0
	},
	select: {
		fontFamily: 'inherit',
		fontSize: '1rem',
		boxSizing: 'border-box',
		marginTop: 0,
		marginBottom: 0,
		lineHeight: 1.75,
		padding: '.5rem .5rem'
	},
	textarea: {
		fontFamily: 'inherit',
		fontSize: '1rem',
		boxSizing: 'border-box',
		marginTop: 0,
		marginBottom: 0,
		lineHeight: 1.75,
		padding: '.5rem .5rem'
	},
	fieldset: {
		fontFamily: 'inherit',
		fontSize: '1rem',
		boxSizing: 'border-box',
		marginTop: 0,
		marginBottom: 0
	},
	label: {
		verticalAlign: 'middle'
	},
	'input[type=text]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=date]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=datetime]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=datetimeLocal]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=email]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=month]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=number]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=password]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=search]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=tel]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=time]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=url]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'input[type=week]': {
		height: '2.25rem',
		padding: '.5rem .5rem',
		verticalAlign: 'middle',
		WebkitAppearance: 'none'
	},
	'select:not([multiple])': {
		height: '2.25rem',
		verticalAlign: 'middle'
	},
	table: {
		borderCollapse: 'separate',
		borderSpacing: 0,
		maxWidth: '100%',
		width: '100%',
		display: 'table'
	},
	th: {
		textAlign: 'left',
		fontWeight: 'bold',
		padding: '.25rem 1rem',
		lineHeight: 'inherit',
		verticalAlign: 'bottom'
	},
	td: {
		padding: '.25rem 1rem',
		lineHeight: 'inherit',
		verticalAlign: 'top'
	},
	h1: {
		fontFamily: '\'Helvetica Neue\', Helvetica, sans-serif',
		fontWeight: 'bold',
		lineHeight: 1.25,
		marginTop: '1em',
		marginBottom: '.5em',
		fontSize: '2rem'
	},
	h2: {
		fontFamily: '\'Helvetica Neue\', Helvetica, sans-serif',
		fontWeight: 'bold',
		lineHeight: 1.25,
		marginTop: '1em',
		marginBottom: '.5em',
		fontSize: '1.5rem'
	},
	h3: {
		fontFamily: '\'Helvetica Neue\', Helvetica, sans-serif',
		fontWeight: 'bold',
		lineHeight: 1.25,
		marginTop: '1em',
		marginBottom: '.5em',
		fontSize: '1.25rem'
	},
	h4: {
		fontFamily: '\'Helvetica Neue\', Helvetica, sans-serif',
		fontWeight: 'bold',
		lineHeight: 1.25,
		marginTop: '1em',
		marginBottom: '.5em',
		fontSize: '1rem'
	},
	h5: {
		fontFamily: '\'Helvetica Neue\', Helvetica, sans-serif',
		fontWeight: 'bold',
		lineHeight: 1.25,
		marginTop: '1em',
		marginBottom: '.5em',
		fontSize: '.875rem'
	},
	h6: {
		fontFamily: '\'Helvetica Neue\', Helvetica, sans-serif',
		fontWeight: 'bold',
		lineHeight: 1.25,
		marginTop: '1em',
		marginBottom: '.5em',
		fontSize: '.75rem'
	},
	p: {
		marginTop: 0,
		marginBottom: '1rem'
	},
	dl: {
		marginTop: 0,
		marginBottom: '1rem'
	},
	ol: {
		marginTop: 0,
		marginBottom: '1rem'
	},
	ul: {
		marginTop: 0,
		marginBottom: '1rem'
	},
	pre: {
		fontFamily: '\'Source Code Pro\', Consolas, monospace',
		fontSize: 'inherit',
		marginTop: 0,
		marginBottom: '1rem',
		overflowX: 'scroll',
		backgroundColor: 'transparent',
		borderRadius: 3
	},
	code: {
		fontFamily: '\'Source Code Pro\', Consolas, monospace',
		fontSize: 'inherit',
		backgroundColor: 'transparent',
		borderRadius: 3
	},
	samp: {
		fontFamily: '\'Source Code Pro\', Consolas, monospace',
		fontSize: 'inherit'
	},
	a: {
		color: '#0074d9',
		textDecoration: 'none',
		':hover': {
			textDecoration: 'underline'
		}
	},
	hr: {
		border: 0,
		borderBottomStyle: 'solid',
		borderBottomWidth: 1,
		borderBottomColor: 'rgba(0,0,0,.125)'
	},
	field: {
		borderStyle: 'solid',
		borderWidth: 1,
		borderColor: 'rgba(0,0,0,.125)',
		borderRadius: 3,
		':focus': {
			outline: 'none',
			borderColor: '#0074d9',
			boxShadow: '0 0 0 2px rgba(0, 116, 217, 0.5)'
		}
	},
	fieldIsFocused: {
		outline: 'none',
		borderColor: '#0074d9',
		boxShadow: '0 0 0 2px rgba(0, 116, 217, 0.5)'
	},
	'field:disabled': {
		backgroundColor: 'rgba(0,0,0,.125)',
		opacity: 0.5
	},
	fieldIsDisabled: {
		backgroundColor: 'rgba(0,0,0,.125)',
		opacity: 0.5
	},
	'field:readOnly:not(select)': {
		backgroundColor: 'rgba(0,0,0,.125)'
	},
	fieldIsReadOnly: {
		backgroundColor: 'rgba(0,0,0,.125)'
	},
	fieldIsSuccess: {
		borderColor: '#2ecc40',
		':focus': {
			boxShadow: '0 0 0 2px rgba(46, 204, 64, 0.5)'
		}
	},
	fieldIsSuccessIsFocused: {
		boxShadow: '0 0 0 2px rgba(46, 204, 64, 0.5)'
	},
	fieldIsWarning: {
		borderColor: '#ffdc00',
		':focus': {
			boxShadow: '0 0 0 2px rgba(255, 220, 0, 0.5)'
		}
	},
	fieldIsWarningIsFocused: {
		boxShadow: '0 0 0 2px rgba(255, 220, 0, 0.5)'
	},
	'field:invalid': {
		borderColor: '#ff4136'
	},
	fieldIsError: {
		borderColor: '#ff4136',
		':focus': {
			boxShadow: '0 0 0 2px rgba(255, 65, 54, 0.5)'
		}
	},
	'field:invalid:focus': {
		boxShadow: '0 0 0 2px rgba(255, 65, 54, 0.5)'
	},
	'field:invalidIsFocused': {
		boxShadow: '0 0 0 2px rgba(255, 65, 54, 0.5)'
	},
	fieldIsErrorIsFocused: {
		boxShadow: '0 0 0 2px rgba(255, 65, 54, 0.5)'
	},
	tableLightTh: {
		borderBottomWidth: 1,
		borderBottomStyle: 'solid',
		borderBottomColor: 'rgba(0,0,0,.125)'
	},
	tableLightTd: {
		borderBottomWidth: 1,
		borderBottomStyle: 'solid',
		borderBottomColor: 'rgba(0,0,0,.125)'
	},
	'tableLightTr:lastChildTd': {
		borderBottom: 0
	},
	btn: {
		fontFamily: 'inherit',
		fontSize: 'inherit',
		fontWeight: 'bold',
		textDecoration: 'none',
		cursor: 'pointer',
		display: 'inline-block',
		lineHeight: '1.125rem',
		padding: '.5rem 1rem',
		margin: 0,
		height: 'auto',
		border: '1px solid transparent',
		verticalAlign: 'middle',
		WebkitAppearance: 'none',
		color: 'inherit',
		backgroundColor: 'transparent',
		':hover': {
			textDecoration: 'none'
		},
		':focus': {
			outline: 'none',
			borderColor: 'rgba(0,0,0,.125)',
			boxShadow: '0 0 0 3px rgba(0,0,0,.25)'
		}
	},
	'::MozFocusInner': {
		border: 0,
		padding: 0
	},
	btnPrimary: {
		color: '#fff',
		backgroundColor: '#0074d9',
		borderRadius: 3,
		':hover': {
			boxShadow: 'inset 0 0 0 20rem rgba(0,0,0,.0625)'
		},
		':active': {
			boxShadow: 'inset 0 0 0 20rem rgba(0,0,0,.125),     inset 0 3px 4px 0 rgba(0,0,0,.25),     0 0 1px rgba(0,0,0,.125)'
		}
	},
	'btnPrimary:disabled': {
		opacity: 0.5
	},
	btnPrimaryIsDisabled: {
		opacity: 0.5
	},
	btnOutline: {
		':hover': {
			borderColor: 'currentcolor',
			boxShadow: 'inset 0 0 0 20rem rgba(0,0,0,.0625)'
		},
		borderRadius: 3,
		':active': {
			boxShadow: 'inset 0 0 0 20rem rgba(0,0,0,.125),     inset 0 3px 4px 0 rgba(0,0,0,.25),     0 0 1px rgba(0,0,0,.125)'
		}
	},
	'btnOutline:disabled': {
		opacity: 0.5
	},
	btnOutlineIsDisabled: {
		opacity: 0.5
	},
	bold: {
		fontWeight: 'bold'
	},
	regular: {
		fontWeight: 'normal'
	},
	italic: {
		fontStyle: 'italic'
	},
	caps: {
		textTransform: 'uppercase',
		letterSpacing: '.2em'
	},
	leftAlign: {
		textAlign: 'left'
	},
	center: {
		textAlign: 'center'
	},
	rightAlign: {
		textAlign: 'right'
	},
	justify: {
		textAlign: 'justify'
	},
	nowrap: {
		whiteSpace: 'nowrap'
	},
	breakWord: {
		wordWrap: 'break-word'
	},
	truncate: {
		maxWidth: '100%',
		overflow: 'hidden',
		textOverflow: 'ellipsis',
		whiteSpace: 'nowrap'
	},
	listReset: {
		listStyle: 'none',
		paddingLeft: 0
	},
	inline: {
		display: 'inline'
	},
	block: {
		display: 'block'
	},
	inlineBlock: {
		display: 'inline-block'
	},
	tableCell: {
		display: 'table-cell'
	},
	overflowHidden: {
		overflow: 'hidden'
	},
	overflowScroll: {
		overflow: 'scroll'
	},
	overflowAuto: {
		overflow: 'auto'
	},
	'clearfix:before': {
		content: '" "',
		display: 'table'
	},
	'clearfix:after': {
		content: '" "',
		display: 'table',
		clear: 'both'
	},
	left: {
		float: 'left'
	},
	right: {
		float: 'right'
	},
	fit: {
		maxWidth: '100%'
	},
	borderBox: {
		boxSizing: 'border-box'
	},
	alignBaseline: {
		verticalAlign: 'baseline'
	},
	alignTop: {
		verticalAlign: 'top'
	},
	alignMiddle: {
		verticalAlign: 'middle'
	},
	alignBottom: {
		verticalAlign: 'bottom'
	},
	m0: {
		margin: 0
	},
	mt0: {
		marginTop: 0
	},
	mr0: {
		marginRight: 0
	},
	mb0: {
		marginBottom: 0
	},
	ml0: {
		marginLeft: 0
	},
	m1: {
		margin: '.5rem'
	},
	mt1: {
		marginTop: '.5rem'
	},
	mr1: {
		marginRight: '.5rem'
	},
	mb1: {
		marginBottom: '.5rem'
	},
	ml1: {
		marginLeft: '.5rem'
	},
	m2: {
		margin: '1rem'
	},
	mt2: {
		marginTop: '1rem'
	},
	mr2: {
		marginRight: '1rem'
	},
	mb2: {
		marginBottom: '1rem'
	},
	ml2: {
		marginLeft: '1rem'
	},
	m3: {
		margin: '2rem'
	},
	mt3: {
		marginTop: '2rem'
	},
	mr3: {
		marginRight: '2rem'
	},
	mb3: {
		marginBottom: '2rem'
	},
	ml3: {
		marginLeft: '2rem'
	},
	m4: {
		margin: '4rem'
	},
	mt4: {
		marginTop: '4rem'
	},
	mr4: {
		marginRight: '4rem'
	},
	mb4: {
		marginBottom: '4rem'
	},
	ml4: {
		marginLeft: '4rem'
	},
	mxn1: {
		marginLeft: '-.5rem',
		marginRight: '-.5rem'
	},
	mxn2: {
		marginLeft: '-1rem',
		marginRight: '-1rem'
	},
	mxn3: {
		marginLeft: '-2rem',
		marginRight: '-2rem'
	},
	mxn4: {
		marginLeft: '-4rem',
		marginRight: '-4rem'
	},
	mxAuto: {
		marginLeft: 'auto',
		marginRight: 'auto'
	},
	p0: {
		padding: 0
	},
	p1: {
		padding: '.5rem'
	},
	py1: {
		paddingTop: '.5rem',
		paddingBottom: '.5rem'
	},
	px1: {
		paddingLeft: '.5rem',
		paddingRight: '.5rem'
	},
	p2: {
		padding: '1rem'
	},
	py2: {
		paddingTop: '1rem',
		paddingBottom: '1rem'
	},
	px2: {
		paddingLeft: '1rem',
		paddingRight: '1rem'
	},
	p3: {
		padding: '2rem'
	},
	py3: {
		paddingTop: '2rem',
		paddingBottom: '2rem'
	},
	px3: {
		paddingLeft: '2rem',
		paddingRight: '2rem'
	},
	p4: {
		padding: '4rem'
	},
	py4: {
		paddingTop: '4rem',
		paddingBottom: '4rem'
	},
	px4: {
		paddingLeft: '4rem',
		paddingRight: '4rem'
	},
	relative: {
		position: 'relative'
	},
	absolute: {
		position: 'absolute'
	},
	fixed: {
		position: 'fixed'
	},
	top0: {
		top: 0
	},
	right0: {
		right: 0
	},
	bottom0: {
		bottom: 0
	},
	left0: {
		left: 0
	},
	z1: {
		zIndex: 1
	},
	z2: {
		zIndex: 2
	},
	z3: {
		zIndex: 3
	},
	z4: {
		zIndex: 4
	},
	smShow: {
		display: 'none',
		'@media (min-width: 40em)': {
			display: 'block'
		}
	},
	mdShow: {
		display: 'none',
		'@media (min-width: 52em)': {
			display: 'block'
		}
	},
	lgShow: {
		display: 'none',
		'@media (min-width: 64em)': {
			display: 'block'
		}
	},
	smHide: {
		'@media (min-width: 40em)': {
			display: 'none'
		}
	},
	mdHide: {
		'@media (min-width: 52em)': {
			display: 'none'
		}
	},
	lgHide: {
		'@media (min-width: 64em)': {
			display: 'none'
		}
	},
	displayNone: {
		display: 'none'
	},
	hide: {
		position: 'absolute',
		height: 1,
		width: 1,
		overflow: 'hidden',
		clip: 'rect(1px, 1px, 1px, 1px)'
	},
	container: {
		maxWidth: '64em',
		marginLeft: 'auto',
		marginRight: 'auto'
	},
	col: {
		float: 'left',
		boxSizing: 'border-box'
	},
	colRight: {
		float: 'right',
		boxSizing: 'border-box'
	},
	col1: {
		width: '8.33333%'
	},
	col2: {
		width: '16.66667%'
	},
	col3: {
		width: '25%'
	},
	col4: {
		width: '33.33333%'
	},
	col5: {
		width: '41.66667%'
	},
	col6: {
		width: '50%'
	},
	col7: {
		width: '58.33333%'
	},
	col8: {
		width: '66.66667%'
	},
	col9: {
		width: '75%'
	},
	col10: {
		width: '83.33333%'
	},
	col11: {
		width: '91.66667%'
	},
	col12: {
		width: '100%'
	},
	smCol: {
		'@media (min-width: 40em)': {
			float: 'left',
			boxSizing: 'border-box'
		}
	},
	smColRight: {
		'@media (min-width: 40em)': {
			float: 'right',
			boxSizing: 'border-box'
		}
	},
	smCol1: {
		'@media (min-width: 40em)': {
			width: '8.33333%'
		}
	},
	smCol2: {
		'@media (min-width: 40em)': {
			width: '16.66667%'
		}
	},
	smCol3: {
		'@media (min-width: 40em)': {
			width: '25%'
		}
	},
	smCol4: {
		'@media (min-width: 40em)': {
			width: '33.33333%'
		}
	},
	smCol5: {
		'@media (min-width: 40em)': {
			width: '41.66667%'
		}
	},
	smCol6: {
		'@media (min-width: 40em)': {
			width: '50%'
		}
	},
	smCol7: {
		'@media (min-width: 40em)': {
			width: '58.33333%'
		}
	},
	smCol8: {
		'@media (min-width: 40em)': {
			width: '66.66667%'
		}
	},
	smCol9: {
		'@media (min-width: 40em)': {
			width: '75%'
		}
	},
	smCol10: {
		'@media (min-width: 40em)': {
			width: '83.33333%'
		}
	},
	smCol11: {
		'@media (min-width: 40em)': {
			width: '91.66667%'
		}
	},
	smCol12: {
		'@media (min-width: 40em)': {
			width: '100%'
		}
	},
	mdCol: {
		'@media (min-width: 52em)': {
			float: 'left',
			boxSizing: 'border-box'
		}
	},
	mdColRight: {
		'@media (min-width: 52em)': {
			float: 'right',
			boxSizing: 'border-box'
		}
	},
	mdCol1: {
		'@media (min-width: 52em)': {
			width: '8.33333%'
		}
	},
	mdCol2: {
		'@media (min-width: 52em)': {
			width: '16.66667%'
		}
	},
	mdCol3: {
		'@media (min-width: 52em)': {
			width: '25%'
		}
	},
	mdCol4: {
		'@media (min-width: 52em)': {
			width: '33.33333%'
		}
	},
	mdCol5: {
		'@media (min-width: 52em)': {
			width: '41.66667%'
		}
	},
	mdCol6: {
		'@media (min-width: 52em)': {
			width: '50%'
		}
	},
	mdCol7: {
		'@media (min-width: 52em)': {
			width: '58.33333%'
		}
	},
	mdCol8: {
		'@media (min-width: 52em)': {
			width: '66.66667%'
		}
	},
	mdCol9: {
		'@media (min-width: 52em)': {
			width: '75%'
		}
	},
	mdCol10: {
		'@media (min-width: 52em)': {
			width: '83.33333%'
		}
	},
	mdCol11: {
		'@media (min-width: 52em)': {
			width: '91.66667%'
		}
	},
	mdCol12: {
		'@media (min-width: 52em)': {
			width: '100%'
		}
	},
	lgCol: {
		'@media (min-width: 64em)': {
			float: 'left',
			boxSizing: 'border-box'
		}
	},
	lgColRight: {
		'@media (min-width: 64em)': {
			float: 'right',
			boxSizing: 'border-box'
		}
	},
	lgCol1: {
		'@media (min-width: 64em)': {
			width: '8.33333%'
		}
	},
	lgCol2: {
		'@media (min-width: 64em)': {
			width: '16.66667%'
		}
	},
	lgCol3: {
		'@media (min-width: 64em)': {
			width: '25%'
		}
	},
	lgCol4: {
		'@media (min-width: 64em)': {
			width: '33.33333%'
		}
	},
	lgCol5: {
		'@media (min-width: 64em)': {
			width: '41.66667%'
		}
	},
	lgCol6: {
		'@media (min-width: 64em)': {
			width: '50%'
		}
	},
	lgCol7: {
		'@media (min-width: 64em)': {
			width: '58.33333%'
		}
	},
	lgCol8: {
		'@media (min-width: 64em)': {
			width: '66.66667%'
		}
	},
	lgCol9: {
		'@media (min-width: 64em)': {
			width: '75%'
		}
	},
	lgCol10: {
		'@media (min-width: 64em)': {
			width: '83.33333%'
		}
	},
	lgCol11: {
		'@media (min-width: 64em)': {
			width: '91.66667%'
		}
	},
	lgCol12: {
		'@media (min-width: 64em)': {
			width: '100%'
		}
	},
	flex: {
		display: 'flex'
	},
	flexColumn: {
		WebkitBoxOrient: 'vertical',
		WebkitBoxDirection: 'normal',
		WebkitFlexDirection: 'column',
		msFlexDirection: 'column',
		flexDirection: 'column'
	},
	flexWrap: {
		WebkitFlexWrap: 'wrap',
		msFlexWrap: 'wrap',
		flexWrap: 'wrap'
	},
	flexCenter: {
		WebkitBoxAlign: 'center',
		WebkitAlignItems: 'center',
		msFlexAlign: 'center',
		alignItems: 'center'
	},
	flexBaseline: {
		WebkitBoxAlign: 'baseline',
		WebkitAlignItems: 'baseline',
		msFlexAlign: 'baseline',
		alignItems: 'baseline'
	},
	flexStretch: {
		WebkitBoxAlign: 'stretch',
		WebkitAlignItems: 'stretch',
		msFlexAlign: 'stretch',
		alignItems: 'stretch'
	},
	flexStart: {
		WebkitBoxAlign: 'start',
		WebkitAlignItems: 'flex-start',
		msFlexAlign: 'start',
		alignItems: 'flex-start'
	},
	flexEnd: {
		WebkitBoxAlign: 'end',
		WebkitAlignItems: 'flex-end',
		msFlexAlign: 'end',
		alignItems: 'flex-end'
	},
	flexJustify: {
		WebkitBoxPack: 'justify',
		WebkitJustifyContent: 'space-between',
		msFlexPack: 'justify',
		justifyContent: 'space-between'
	},
	flexAuto: {
		WebkitBoxFlex: 1,
		WebkitFlex: '1 1 auto',
		msFlex: '1 1 auto',
		flex: '1 1 auto',
		minWidth: 0,
		minHeight: 0
	},
	flexGrow: {
		WebkitBoxFlex: 1,
		WebkitFlex: '1 0 auto',
		msFlex: '1 0 auto',
		flex: '1 0 auto'
	},
	flexNone: {
		WebkitBoxFlex: 0,
		WebkitFlex: 'none',
		msFlex: 'none',
		flex: 'none'
	},
	flexFirst: {
		WebkitBoxOrdinalGroup: 0,
		WebkitOrder: -1,
		msFlexOrder: -1,
		order: -1
	},
	flexLast: {
		WebkitBoxOrdinalGroup: 100000,
		WebkitOrder: 99999,
		msFlexOrder: 99999,
		order: 99999
	},
	smFlex: {
		'@media (min-width: 40em)': {
			display: 'flex'
		}
	},
	mdFlex: {
		'@media (min-width: 52em)': {
			display: 'flex'
		}
	},
	lgFlex: {
		'@media (min-width: 64em)': {
			display: 'flex'
		}
	},
	border: {
		borderStyle: 'solid',
		borderWidth: 1,
		borderColor: 'rgba(0,0,0,.125)'
	},
	borderTop: {
		borderTopStyle: 'solid',
		borderTopWidth: 1,
		borderTopColor: 'rgba(0,0,0,.125)'
	},
	borderRight: {
		borderRightStyle: 'solid',
		borderRightWidth: 1,
		borderRightColor: 'rgba(0,0,0,.125)'
	},
	borderBottom: {
		borderBottomStyle: 'solid',
		borderBottomWidth: 1,
		borderBottomColor: 'rgba(0,0,0,.125)'
	},
	borderLeft: {
		borderLeftStyle: 'solid',
		borderLeftWidth: 1,
		borderLeftColor: 'rgba(0,0,0,.125)'
	},
	borderNone: {
		border: 0
	},
	rounded: {
		borderRadius: 3
	},
	circle: {
		borderRadius: '50%'
	},
	roundedTop: {
		borderRadius: '3px 3px 0 0'
	},
	roundedRight: {
		borderRadius: '0 3px 3px 0'
	},
	roundedBottom: {
		borderRadius: '0 0 3px 3px'
	},
	roundedLeft: {
		borderRadius: '3px 0 0 3px'
	},
	notRounded: {
		borderRadius: 0
	},
	black: {
		color: '#111'
	},
	gray: {
		color: '#aaa'
	},
	silver: {
		color: '#ddd'
	},
	white: {
		color: '#fff'
	},
	aqua: {
		color: '#7fdbff'
	},
	blue: {
		color: '#0074d9'
	},
	navy: {
		color: '#001f3f'
	},
	teal: {
		color: '#39cccc'
	},
	green: {
		color: '#2ecc40'
	},
	olive: {
		color: '#3d9970'
	},
	lime: {
		color: '#01ff70'
	},
	yellow: {
		color: '#ffdc00'
	},
	orange: {
		color: '#ff851b'
	},
	red: {
		color: '#ff4136'
	},
	fuchsia: {
		color: '#f012be'
	},
	purple: {
		color: '#b10dc9'
	},
	maroon: {
		color: '#85144b'
	},
	colorInherit: {
		color: 'inherit'
	},
	muted: {
		opacity: 0.5
	},
	bgBlack: {
		backgroundColor: '#111'
	},
	bgGray: {
		backgroundColor: '#aaa'
	},
	bgSilver: {
		backgroundColor: '#ddd'
	},
	bgWhite: {
		backgroundColor: '#fff'
	},
	bgAqua: {
		backgroundColor: '#7fdbff'
	},
	bgBlue: {
		backgroundColor: '#0074d9'
	},
	bgNavy: {
		backgroundColor: '#001f3f'
	},
	bgTeal: {
		backgroundColor: '#39cccc'
	},
	bgGreen: {
		backgroundColor: '#2ecc40'
	},
	bgOlive: {
		backgroundColor: '#3d9970'
	},
	bgLime: {
		backgroundColor: '#01ff70'
	},
	bgYellow: {
		backgroundColor: '#ffdc00'
	},
	bgOrange: {
		backgroundColor: '#ff851b'
	},
	bgRed: {
		backgroundColor: '#ff4136'
	},
	bgFuchsia: {
		backgroundColor: '#f012be'
	},
	bgPurple: {
		backgroundColor: '#b10dc9'
	},
	bgMaroon: {
		backgroundColor: '#85144b'
	},
	bgDarken1: {
		backgroundColor: 'rgba(0,0,0,.0625)'
	},
	bgDarken2: {
		backgroundColor: 'rgba(0,0,0,.125)'
	},
	bgDarken3: {
		backgroundColor: 'rgba(0,0,0,.25)'
	},
	bgDarken4: {
		backgroundColor: 'rgba(0,0,0,.5)'
	},
	bgLighten1: {
		backgroundColor: 'rgba(255,255,255,.0625)'
	},
	bgLighten2: {
		backgroundColor: 'rgba(255,255,255,.125)'
	},
	bgLighten3: {
		backgroundColor: 'rgba(255,255,255,.25)'
	},
	bgLighten4: {
		backgroundColor: 'rgba(255,255,255,.5)'
	}
}
